src/Symfony/Bundle/FrameworkBundle/Resources/config/fragment_renderer.xml

```diff
@@ -12,8 +12,8 @@
     <services>
         <service id="fragment.handler" class="Symfony\Component\HttpKernel\DependencyInjection\LazyLoadingFragmentHandler">
             <argument type="service" id="service_container" />
-            <argument>%kernel.debug%</argument>
             <argument type="service" id="request_stack" />
+            <argument>%kernel.debug%</argument>
```

Comment:
I don't get it. There is a BC break anyway. In 2.x the RequestStack was optional. But in 3.0 it is required. The argument order is just a tiny detail. The BC break is there either way.